docs/developers/guides/run-a-node/linea-sequencer.mdx

```diff
@@ -0,0 +1,30 @@
+---
+title: Install the linea-sequencer plugin for BESU
+description: How to install and configure the linea-sequencer plugin for Besu
+image: /img/socialCards/install-the-linea-sequencer-plugin-for-besu.jpg
+---
+
+# Install `linea-sequencer`
```

Comment:
We may need to specifically mention somewhere that the sequencer will run with Besu only.